test/extensions/access_loggers/wasm/config_test.cc

```diff
@@ -33,7 +34,8 @@ class WasmAccessLogConfigTest
   WasmAccessLogConfigTest() : api_(Api::createApiForTest(stats_store_)) {
     ON_CALL(context_.server_factory_context_, api()).WillByDefault(ReturnRef(*api_));
     ON_CALL(context_, scope()).WillByDefault(ReturnRef(scope_));
-    ON_CALL(context_, listenerMetadata()).WillByDefault(ReturnRef(listener_metadata_));
+    ON_CALL(context_, listenerInfo()).WillByDefault(ReturnRef(listener_info_));
+    ON_CALL(listener_info_, metadata()).WillByDefault(ReturnRef(listener_metadata_));
```

Comment:
`ON_CALL(context_.listener_info_, ...);` is enough. We needn't create a local listener info again.